python/mjai/bot/base.py

```diff
@@ -212,10 +212,13 @@ def tehai_mjai(self) -> list[str]:
         for tile_idx, tile_count in enumerate(self.player_state.tehai):
             if tile_count and tile_idx == 4 and self.akas_in_hand[0]:
                 akas.append("5mr")
+                ms += [f"{tile_idx + 1}m"] * tile_count
```

Comment:
When I have three 5m and one 5mr in hand, the tile_count will be 4.
that results: tiles = [..."5m", "5m", "5m", "5m", "5mr",...]

Reply:
I resolved it on this issue. The current code reduces the tile count of 5{m,p,s} when the corresponding akadora is in the hand.
https://github.com/smly/mjai.app/blob/472f1592cec074d5a58f83c4571e56d93ce9f3a6/python/mjai/bot/base.py#L210-L213

You can check the results of three 5m and one 5mr in hand by adding the following test case to test_bot.py.

```py
    # Case6
    bot.player_state.tehai = list(
        map(int, list("0111411110000100000000100000000000"))
    )
    bot.player_state.akas_in_hand = [True, False, True]
    assert bot.tehai == "23405556789m5p0s"
    assert bot.tehai_mjai.count("5m") == 3
    assert bot.tehai_mjai.count("5mr") == 1
```